.github/workflows/Code Coverage.yml

```diff
@@ -0,0 +1,59 @@
+name: CI Pipeline Code Coverage
+
+on:
+  pull_request:
+    branches: [ main ]
+
+  # Allow manually triggering from the actions tab
+  workflow_dispatch:
+
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Debug
+
+  # Enable colour output for GTEST and XML file generation
+  GTEST_COLOR: yes
+  GTEST_OUTPUT: xml:test-results/
+
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic anda should work equally well on Windows or Mac.
+    # You can convert this to a matrix build if you need cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install OpenCppCoverage
+      uses: KungFuDonkey/Install-OpenCppCoverage@v1.0.1
+
+    - name: Configure CMake
+      run: cmake -DOPTION_GITHUB_ACTIONS=ON -S ${{github.workspace}}/SDALib-CPP -B ${{runner.workspace}}/build -T host=x86 -A Win32
+
+    - name: Build
+      working-directory: ${{runner.workspace}}/build
+      run: cmake --build .
+
+    - name: Run code coverage
+      working-directory: ${{github.workspace}}
+      run: |
+        cd SDALib-CPP
+        .\SDACoverage.bat
```

Comment:
The file is named `coverage.bat` in IPCLib, either make this one `coverage.bat` or the other one `IPCLibCoverage.bat` for consistance